src/interfaces/IPDPVerifier.sol

```diff
@@ -40,4 +40,8 @@ interface IPDPVerifier is IPDPEvents {
         external
         view
         returns (IPDPTypes.PieceIdAndOffset[] memory);
+
+    // Fee update functions
+    function updateProofFee(uint256 newFeePerTiB) external;
+    function applyFeeUpdate() external;
```

Comment:
these don't belong in the IPDPVerifier interface. can make another interface if you want to standardize it but I don't think that's necessary either

Reply:
`feePerTiB` probably does belong in the interface though